src/components/Modals/FiatRamps/views/FiatForm.tsx

```diff
@@ -109,6 +109,7 @@ export const FiatForm: React.FC<FiatFormProps> = ({
           const { chainId } = fromAccountId(accountId)
           const maybeAdapter = getChainAdapterManager().get(chainId)
           if (!maybeAdapter) return Promise.resolve(`no chain adapter for ${chainId}`)
+          // @ts-ignore, we actually only want to bypass on-device derivation for Ledger here
```

Comment:
We can improve safety here by removing the `ts-ignore` and above doing:

```ts
const payload = {
            accountType,
            accountNumber,
            wallet,
            ...(isLedger(wallet)
              ? { checkLedgerAppOpenIfLedgerConnected: () => Promise.resolve() }
              : { pubKey: fromAccountId(accountId).account }),
          }
```

Reply:
https://github.com/shapeshift/web/commit/e08c8e4ce818c1929b14888d9ed3e48799a41e14

Reply:
a61fcfdad4